test/fixtures/aws-component-helper/atmos.yaml

```
@@ -0,0 +1,86 @@
+      enabled: true
+    # https://docs.gomplate.ca
+    gomplate:
+      enabled: true
+
+commands:
+  - name: "test-components"
+    description: "List the Atmos virtual components configured for testing"
+    steps:
+      - >
+        atmos describe stacks --format json --sections=component,metadata --components=component -s sandbox
+        | jq '.[] | .components.terraform | to_entries | 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)